### PR TITLE
Fix the tests and update the supported python/django versions

### PR DESCRIPTION
**Pull request by MDziwny**
I've fixed the tests, added new versions of python (3.7) and django (2.0 and 2.1). I've also removed deprecated versions of python (2.6 and 3.3) and django (1.6, 1.7, 1.8 and 1.10).

If you want to merge it into your master branch, I can put back your Travis and Gitter icons.

### CODE REVIEW

**Comment by urkh** [comment]
What is the status of this project?

**Comment by MDziwny** [comment]
I've modified my PR according to the comments.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+django-fsm 2.6.1 2018-12-14
```
Comment (imomaliev) [inline CHANGELOG.rst]: Forgot to update version to `2.7.0` here

Comment (MDziwny) [reply CHANGELOG.rst]: I've added the 2.7.0 part in the changelog.

**Comment by imomaliev** [comment]
@kmmbvnr when this will be merged?

**Comment by nuthingman** [comment]
maybe someone should just fork this?

**Comment by MDziwny** [comment]
I'm thinking about forking it and publishing it to pypi but it would be better to contribute to this project :(

**Comment by urkh** [comment]
@MDziwny I support you, I think this project is abandoned

**Comment by kmmbvnr** [comment]
Sorry for the delay. I've been working on other projects.

Than you for PR!

**Pull request closed without merging**